network/internal/testutils/testUtil.go

```diff
@@ -91,13 +92,17 @@ func (cwcm *TagWatchingConnManager) Unprotect(id peer.ID, tag string) bool {
 	return res
 }
 
-func NewTagWatchingConnManager(log zerolog.Logger, metrics module.LibP2PConnectionMetrics) *TagWatchingConnManager {
-	cm := connection.NewConnManager(log, metrics)
+func NewTagWatchingConnManager(log zerolog.Logger, metrics module.LibP2PConnectionMetrics, config *connection.ManagerConfig) (*TagWatchingConnManager, error) {
```

Comment:
Add a godoc comment describing the TagWatchingConnManager and it's purpose.

Reply:
https://github.com/onflow/flow-go/pull/3852/commits/bcc877c917f3f28fb6d0680ece02812c4bda50ef

Reply:
https://github.com/onflow/flow-go/pull/3852/commits/bcc877c917f3f28fb6d0680ece02812c4bda50ef